src/xmpp/iq.c

```diff
@@ -308,6 +309,21 @@ iq_disco_info_request(gchar *jid)
 }
 
 void
+iq_disco_info_request_onconnect(gchar *jid)
+{
+    xmpp_ctx_t * const ctx = connection_get_ctx();
+    char *id = create_unique_id("disco_info_onconnect");
```

Comment:
Indeed, there's loads of work to do on Stabber, it's really just a start as I couldn't find anything out there to run tests against.